trunk/SUAVE/Components/Energy/Networks/Lift_Cruise.py

```diff
@@ -133,257 +134,375 @@ def evaluate_thrust(self,state):
-        volts[volts>self.voltage] = self.voltage 
-        
-        # ESC Voltage
-        lift_rotor_esc.inputs.voltagein = volts      
-        propeller_esc.inputs.voltagein  = volts 
-        
-        #---------------------------------------------------------------
-        # EVALUATE THRUST FROM FORWARD PROPULSORS 
-        #---------------------------------------------------------------
-        # Throttle the voltage
-        propeller_esc.voltageout(conditions) 
-        
-        # How many evaluations to do
-        if self.identical_propellers:
-            n_evals = 1
-            factor  = num_forward*1
-        else:
-            n_evals = int(num_forward)
-            factor  = 1.
-        
-        # Setup numbers for iteration
-        total_prop_motor_current = 0.
-        total_prop_thrust        = 0. * state.ones_row(3)
-        total_prop_power         = 0.
+        battery.current_energy      = conditions.propulsion.battery_energy
+        battery.pack_temperature    = conditions.propulsion.battery_pack_temperature
+        battery.charge_throughput   = conditions.propulsion.battery_charge_throughput     
+        battery.age_in_days         = conditions.propulsion.battery_age_in_days 
+        discharge_flag              = conditions.propulsion.battery_discharge    
+        battery.R_growth_factor     = conditions.propulsion.battery_resistance_growth_factor
+        battery.E_growth_factor     = conditions.propulsion.battery_capacity_fade_factor 
+        battery.max_energy          = conditions.propulsion.battery_max_aged_energy
+        V_th0                       = conditions.propulsion.battery_initial_thevenin_voltage
+        n_series                    = battery.pack_config.series  
+        n_parallel                  = battery.pack_config.parallel
+        n_total                     = n_series*n_parallel
+        
+        # update ambient temperature based on altitude
+        battery.ambient_temperature                   = conditions.freestream.temperature   
+        battery.cooling_fluid.thermal_conductivity    = conditions.freestream.thermal_conductivity
+        battery.cooling_fluid.kinematic_viscosity     = conditions.freestream.kinematic_viscosity
+        battery.cooling_fluid.density                 = conditions.freestream.density  
+        battery.ambient_pressure                      = conditions.freestream.pressure  
+        a                                             = conditions.freestream.speed_of_sound
         
-        # Iterate over motor/props
-        for ii in range(n_evals):    
+        # Set battery energy
+        battery.current_energy = conditions.propulsion.battery_energy  
+
+
+        # --------------------------------------------------------------------------------
+        # Predict Voltage and Battery Properties Depending on Battery Chemistry
+        # --------------------------------------------------------------------------------  
```

Comment:
Much of this is the exact same updates that were made for the Battery_Propeller.py; let's combine to reduce repeated code

Reply:
Reduced into callable functions